script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -650,13 +660,13 @@ Elegantly, without the slightest sway of her long, black hair.
 [sha:bcb3c480dbe57bc730dba86e597e077c7bddae12]{
 -- Page 36, Offset 29095.
 -- 品のある立ち振る舞いに見惚れてしまう。
-I am entranced by her dignified bearing.
+I am entranced by her stately bearing.
 }
 [sha:ebe0e79328c6238e33d40e87fa46cb2d663d82b2]{
 -- Page 36, Offset 29096.
 -- 我が事ながら自分が分からない。
 // these two lines are quite liberal, but closer to intent
-I know this is myself I'm talking about, but what have I even been doing?
+I'm starting to wonder if there might be something wrong with me.
```

Comment:
我が事ながら feels like it needs to be preserved in some creative way, since it gives the contrast of Roa/Shiki's thoughts.

>What's wrong with you, me?

Something like that.